_posts/2018-08-01-umbrellas-just-when-it-rains.md

```diff
@@ -0,0 +1,84 @@
+---
+author: Sean Callan
+author_link: https://github.com/doomspork
+categories: general
+date: 2018-08-01
+ayout: post
+title: Umbrellas: only when it rains?
+excerpt: >
+  A look at Umbrella applications and how they can help us write cleaner maintainable code.
+---
+
+The Umbrella application is powerful tool available to us but many people are unfamiliar with them.
```

Comment:
Something wrong here. maybe `is a powerful tool`, but check at the ending of the sentence that you are referring to Umbrella in plural (them).